AU-Policy.md

```diff
@@ -101,7 +101,7 @@ We use reporting rulesets developed by the Snort, Nessus and ClamAV teams, which
 Security vulnerabilities and system inconsistencies are reviewed by the Cloud Operations team (notified by email, text message and voice phone call).  Security vulnerabilities which are not classified as high are reviewed weekly and resolved by Cloud Operations. Regular security reports are automatically generated by Nessus and sent to the System Owner, GSA’s Information Security team and other partner agencies as required.
 See SI procedures for more detail.
 
-The Cloud Operations team acts on findings that result from its regular audit process according to its incident response guidelines (https://docs.cloud.gov/ops/security-ir), including notifying GSA Information Security, the System Owner, and the ISSO.
+The Cloud Operations team acts on findings that result from its regular audit process according to its incident response guidelines (https://cloud.gov/docs/ops/security-ir/), including notifying GSA Information Security, the System Owner, and the ISSO.
```

Comment:
Hi Chiaka, looks like this link needs to be updated after the team docs were moved